redisinsight/ui/src/helpers/constructKeysToTree.ts

```diff
@@ -2,21 +2,19 @@ import { IKeyPropTypes } from 'uiSrc/constants/prop-types/keys'
 
 interface Props {
   items: IKeyPropTypes[]
-  separator?: string
+  delimiter?: string
 }
 
 export const constructKeysToTree = (props: Props): any[] => {
-  const { items: keys, separator = ':' } = props
-  // const keysSymbol = Symbol(`keys${separator}keys`)
-  // const keysSymbol = Symbol('keys')
-  const keysSymbol = `keys${separator}keys`
+  const { items: keys, delimiter = ':' } = props
+  const keysSymbol = `keys${delimiter}keys`
```

Comment:
maybe its better to use getTreeLeafField here

Reply:
This is file for WebWorker and I can't import getTreeLeafField function inside WebWorker